include/irr/asset/format/convertColor.h

```diff
@@ -159,8 +159,17 @@ namespace irr { namespace video
         }
 	#undef SWIZZLE
     }
+	/*
+		It converts a data to desired texel format
+
+		To use it, you have to pass source data with texel format \bsF\b you want to exchange
+		with \bdF\b. srcPix data is an array due to planar formats, but if source data isn't
+		a planar format, you can use \bsrcPix[0]\b.
+		\b_pixOrBlockCnt\b is an amount of texels you want to convert and \b_imgSize\b is a size
+		in texels of an image they belong to.
+	*/
```

Comment:
polymorphic swizzle is a class that can swizzle the RGBA componenents into a different arrangement at runtime, using it (non null pointer) obviously makes the whole process slower.